frontend/src/components/__tests__/FileUpload.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import FileUpload from '../FileUpload';
+import { render, cleanup } from 'react-testing-library';
+import 'jest';
+import 'jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+describe('FileUpoad', () => {
+  test('should render a Button', () => {
+    const { container } = render(<FileUpload />);
+    const buttons = container.querySelector('div');
+
+    expect(buttons).toBeInTheDocument();
+  });
+
+  test('should render passed text', () => {
+    const { container } = render(<FileUpload text='this is a thing' />);
+    const buttons = container.querySelector('button');
+    expect(buttons.textContent).toBe('this is a thing');
```

Comment:
I believe we should at least be able to confirm that the modal is opening.
If you're unsure how to do that, and how the DOM changes when clicking the button you can use `debug()` that comes from the render function

Reply:
Working on it now